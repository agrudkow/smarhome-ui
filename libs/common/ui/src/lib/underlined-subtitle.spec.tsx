import React from 'react';
import { render } from '@testing-library/react';

import UnderlinedSubtitle from './underlined-subtitle';

describe(' UnderlinedSubtitle', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<UnderlinedSubtitle />);
    expect(baseElement).toBeTruthy();
  });
});
