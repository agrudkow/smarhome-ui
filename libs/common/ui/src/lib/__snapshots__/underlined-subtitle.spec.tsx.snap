// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` UnderlinedSubtitle should match snapshot 1`] = `
.c0 {
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  margin-top: 15px;
  opacity: 0.8;
  border-bottom: 2px solid #4E4E4E;
}

@media (max-width:769px) {
  .c0 {
    width: 100%;
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
  }
}

<div>
  <div
    class="c0"
  />
</div>
`;
